Makefile

```diff
@@ -58,6 +58,11 @@ init-package: ## Create the zarf init package, macos "brew install coreutils" fi
 
 build-test: build-cli init-package ## Build the CLI and create the init package
 
+publish-image:
+	docker buildx build --push --platform linux/arm64/v8,linux/amd64 --tag defenseunicorns/zarf-agent:${TAG} .
```

Comment:
This `${TAG}` is from an env right? When it fails right now there's no clear message that you need to export some. value to `TAG` unless you go look at the Makefile. Is an error message worth it to you?

Reply:
It's a makefile arg and already passed in the manual trigger?